astroplan/core.py

```diff
@@ -0,0 +1,490 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+#from ..extern import six
+
+#import sys
+#from math import sqrt, pi, exp, log, floor
+from abc import ABCMeta, abstractmethod
+
+#import numpy as np
+
+# Originally authored by Jazmin Berlanga Medina (jazmin.berlanga@gmail.com),
+# and modified by Adrian Price-Whelan (email) and Erik Tollerud (email).
+
+__all__ = ["Observer", "Target", "FixedTarget", "NonFixedTarget",
+           "Constraint", "TimeWindow", "AltitudeRange",
+           "AboveAirmass", "Observation"]
+
+#__doctest_requires__ = {'*': ['scipy.integrate']}
+
+
+class Observer(object):
+    """
+    Some comments.
+    """
+
+    def __init__(self, name, longitude, latitude, elevation, pressure,
```

Comment:
`longitude` and `latitude` should simply be `location`, and this should expect an `astropy.coordinates.EarthLocation` instance. Also, not critical for this skeleton, but we'll probably want to allow someone to instantiate an `Observer` with just a name, location, and timezone (a topic for discussion is whether we allow `timezone=None` and just infer the timezone? I'll make a note to bring this up next telecon...)